src/orchard/meta.clj

```diff
@@ -234,14 +234,18 @@
               collect? (atom false)
               pbr      (proxy [LineNumberingPushbackReader] [rdr]
                          (read []
-                           (let [i (proxy-super read)]
+                           (let [;; remove reflection warning on proxy-super call
```

Comment:
I guess inline comments would be better, as they don't break the code flow here.

Reply:
I put this on the line above to avoid a long line (120 chars), however, if you prefer them online I'll fix them.